.travis.yml

```diff
@@ -21,7 +21,7 @@ before_install:
   - pip install python-dateutil>=2.6.0
   - pip install pandas==0.18.1
   - pip install requests>=2.9
-  - pip install httpretty==0.8.6
+  - pip install httpretty
```

Comment:
Interesting. This means that we no longer need to specify `httpretty==0.8.6`? If that's the case, shouldn't we specify `>=something`?

Reply:
Yes, you're probably right. We can use the latest version of `httpretty` (0.9.5).

Reply:
In that case, I suggest that you change that line to

`pip install httpretty>=0.9.5`